.circleci/config.yml

```diff
@@ -57,8 +57,9 @@ setup_fx2ait_env: &setup_fx2ait_env
           sudo cp -P cudnn-*-archive/lib/libcudnn* /usr/local/cuda/lib64
           sudo chmod a+r /usr/local/cuda/include/cudnn*.h /usr/local/cuda/lib64/libcudnn*
           python3.8 -m pip install --ignore-installed --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu118
-          python3.8 fx2ait/setup.py install --prefix=/home/circleci/
-          echo 'export PYTHONPATH=$PWD/fx2ait:$PYTHONPATH' >> $BASH_ENV
+          pushd fx2ait
+          python3.8 setup.py develop --user
+          popd
```

Comment:
Curious why do we need to change the CI config?

Reply:
The original command `python3.8 fx2ait/setup.py install --prefix=/home/circleci/` actually installed the package to `/home/circleci/lib/python3.8/site-packages/fx2ait-0.2.dev1-py3.9-linux-x86_64.egg/fx2ait` and won't be automatically importable unless you also set `PYTHONPATH` to it.

Another issue I found is `/home/project/fx2ait` will be automatically added to `PYTHONPATH` (due to `/home/project/fx2ait/fx2ait/test/__init__.py` file), so you are actually importing `/home/project/fx2ait/fx2ait` when running the test. Using `--develop` is to make sure you have `/home/project/fx2ait/fx2ait/libait_model.so` installed correctly.